packages/components/Button/src/FAB/FABColorTokens.ts

```diff
@@ -1,10 +1,12 @@
 import { Theme } from '@fluentui-react-native/framework';
 import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { ButtonCoreTokens } from '../Button.types';
-import { shadowStyleFromTheme } from '../shadowStyle';
+import { shadowStyleFromTheme } from '@fluentui-react-native/experimental-shadow';
 
 export const defaultFABColorTokens: TokenSettings<ButtonCoreTokens, Theme> = (t: Theme): ButtonCoreTokens => ({
-  ...shadowStyleFromTheme(t, 'shadow8'),
+  // Grab the second shadow from the global token, as we only support displaying one shadow
+  ...shadowStyleFromTheme(t, '8').key,
+
```

Comment:
As mentioned offline, I think it would be best to revert this change, duplicate the file, and also delete the change file for experimental-button

Reply:
Just updated